monodoc/monodoc.csproj

```diff
@@ -1,713 +1,27 @@
-      <LogicalName>protinterface.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protmethod.png">
-      <LogicalName>protmethod.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protproperty.png">
-      <LogicalName>protproperty.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protstructure.png">
-      <LogicalName>protstructure.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubclass.png">
-      <LogicalName>pubclass.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubdelegate.png">
-      <LogicalName>pubdelegate.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubenumeration.png">
-      <LogicalName>pubenumeration.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubevent.png">
-      <LogicalName>pubevent.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubextension.png">
-      <LogicalName>pubextension.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubfield.png">
-      <LogicalName>pubfield.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubinterface.png">
-      <LogicalName>pubinterface.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubmethod.png">
-      <LogicalName>pubmethod.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubproperty.png">
-      <LogicalName>pubproperty.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubstructure.png">
-      <LogicalName>pubstructure.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/reference.png">
-      <LogicalName>reference.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/treebg.png">
-      <LogicalName>treebg.png</LogicalName>
-    </EmbeddedResource>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="monodoc.dll.config">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <PackageReference Include="SharpZipLib" Version="1.0.0-alpha2" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.4.1" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\external\SharpZipLib\ICSharpCode.SharpZipLib.NET45\ICSharpCode.SharpZipLib.csproj">
-      <Project>{0e7413ff-eb9e-4714-acf2-be3a6a7b2ffd}</Project>
-      <Name>ICSharpCode.SharpZipLib</Name>
-    </ProjectReference>
+    <Compile Include="..\external\Lucene.Net.Light\src\core\**\*.cs" Link="Lucene\%(Filename)%(Extension)" />
+    <Compile Remove="Mono.Utilities\MemoryLRU.cs" />

```

Comment:
These files needed to be explicitly removed from the wildcard to match the old project's explicit file list